sql/moz-fx-data-shared-prod/firefox_accounts_derived/fxa_delete_events_v2/query.sql

```diff
@@ -0,0 +1,30 @@
+WITH hmac_key AS (
+  SELECT
+    AEAD.DECRYPT_BYTES(
+      (SELECT keyset FROM `moz-fx-dataops-secrets.airflow_query_keys.fxa_prod`),
+      ciphertext,
+      CAST(key_id AS BYTES)
+    ) AS value
+  FROM
+    `moz-fx-data-shared-prod.firefox_accounts_derived.encrypted_keys_v1`
+  WHERE
+    key_id = 'fxa_hmac_prod'
+)
+SELECT
+  `timestamp` AS submission_timestamp,
+  TO_HEX(SHA256(jsonPayload.fields.uid)) AS user_id,
+  TO_HEX(
+    udf.hmac_sha256((SELECT * FROM hmac_key), CAST(jsonPayload.fields.uid AS BYTES))
+  ) AS hmac_user_id,
+FROM
+  `moz-fx-fxa-prod.gke_fxa_prod_log.stderr`
```

Comment:
@sean-rose Are we sure all deletion requests arrive via the `stderr` table and not also via the `stdout` table?

Reply:
Yes, the delete events are coming from the FxA auth server, which currently has its logs landing in the `stderr` table.

FxA does plan to standardize their logging to consistently send all logs to either `stderr` or `stdout`, and if they pick `stdout` then at that point we'd need to add a v3 ETL that selects from `stdout` instead and include that in the unioning view.